ceph/ceph_admin/osd.py

```diff
@@ -106,3 +106,106 @@ def apply(self, config: Dict) -> None:
             sleep(interval)
 
         raise OSDServiceFailure("OSDs are not up and running in hosts")
+
+    def rm_status(self, config):
+        """
```

Comment:
Since our API documentation used Rst format, please follow doc string format as mentioned in this method,
https://github.com/red-hat-storage/cephci/blob/master/ceph/ceph_admin/daemon.py#L13-L31

Reply:
Addressed the comments and modified doc string as per the format

Reply:
Addressed the comments as per the format given above.